arraycontext/pytest.py

```diff
@@ -257,6 +264,15 @@ def pytest_generate_tests_for_pyopencl_array_context(metafunc) -> None:
     for device selection.
     """
 
+    from warnings import warn
+    warn("pytest_generate_tests_for_pyopencl_array_context is deprecated. "
+            "Use 'pytest_generate_tests = "
+            "arraycontext.pytest_generate_tests_for_array_contexts"
+            "[\"pyopencl-deprecated\"]' instead. "
```

Comment:
```suggestion
            "([\"pyopencl-deprecated\"])' instead. "
```

Reply:
Done as part of this force-push: https://github.com/inducer/arraycontext/compare/a68d80547bf07de6d209ed756aafbb743dcda2b4..7a8cf8aeb0cf5b0c7f579dea5634d8a3dc70abf0